kernel/comps/systree/src/node.rs

```diff
@@ -16,9 +16,9 @@ pub const MAX_ATTR_SIZE: usize = 4096;
 /// The three types of nodes in a `SysTree`.
 #[derive(Copy, Clone, Debug, PartialEq, Eq)]
 pub enum SysNodeType {
-    /// A branching node is one that may contain child nodes.
+    /// A branching node is one that are able to contain child nodes.
```

Comment:
Grammar: are --> is

"be able to" is too verbose. I suggest rewriting the two sentences for branching nodes and leaf nodes with _can_.

```
/// A branching node is one that can have child nodes.

/// A leaf node is one that cannot have child nodes.
```